app/_src/kubernetes-ingress-controller/guides/high-availability/fallback-config.md

```diff
@@ -0,0 +1,327 @@
+      port: 1027
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: route-b
+spec:
+  parentRefs:
+  - name: kong
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /route-b
+    backendRefs:
+    - name: echo
+      port: 1027
+      ' | kubectl apply -f -
+```
+
+The results should look like this:
+```text
+httproute.gateway.networking.k8s.io/route-a created
+httproute.gateway.networking.k8s.io/route-b created
+```
+
+Let's ensure that the `HTTPRoutes` are working as expected:
+
+```bash
+curl -i $PROXY_IP/route-a
+```
+
+The results should look like this:
+```text
+HTTP/1.1 200 OK
+Content-Type: text/plain; charset=utf-8
+Content-Length: 137
+Connection: keep-alive
+X-Kong-Upstream-Latency: 1
+X-Kong-Proxy-Latency: 1
+Via: kong/3.6.0
+X-Kong-Request-Id: 5bf50016730eae43c359c17b41dc8614
+
+Welcome, you are connected to node orbstack.
+Running on Pod echo-74c66b778-szf8f.
+In namespace default.
+With IP address 192.168.194.13.
+```
+
+```bash
+curl -i $PROXY_IP/route-b
+```
+
+The results should look like this:
+```text
+HTTP/1.1 200 OK
+Content-Type: text/plain; charset=utf-8
+Content-Length: 137
+Connection: keep-alive
+X-Kong-Upstream-Latency: 0
+X-Kong-Proxy-Latency: 0
+Via: kong/3.6.0
+X-Kong-Request-Id: 14ae28589baff9459d5bb3476be6f570
+
+Welcome, you are connected to node orbstack.
+Running on Pod echo-74c66b778-szf8f.
+In namespace default.
+With IP address 192.168.194.13.
+```
+
+Now, let's introduce a breaking change in the `route-2` by attaching a broken `KongPlugin` to it.
+
+Create a `KongPlugin` with an invalid configuration:
+
+```bash
+echo 'apiVersion: configuration.konghq.com/v1
+kind: KongPlugin
+metadata:
+  name: key-auth
+plugin: key-auth
+config:
+  # Should be 'key_names', not 'keys'.
+  keys: ["key"]' | kubectl apply -f -
+```
+
+The results should look like this:
+```text
+kongplugin.configuration.konghq.com/key-auth created
+```
+
+Now, let's attach the `KongPlugin` to `route-b`:
+
+```bash
+kubectl annotate httproute route-b konghq.com/plugins=key-auth
+```
+
+The results should look like this:
+```text
+httproute.gateway.networking.k8s.io/route-b patched
+```
+
+This will cause the `route-b` to break. Let's verify this:
+
+```bash
+curl -i $PROXY_IP/route-b
+```
+
+The results should look like this:
+```text
+HTTP/1.1 404 Not Found
+Content-Type: application/json; charset=utf-8
+Connection: keep-alive
+Content-Length: 103
+X-Kong-Response-Latency: 0
+Server: kong/3.6.0
+X-Kong-Request-Id: 209a6b14781179103528093188ed4008
+
+{
+  "message":"no Route matched with those values",
+  "request_id":"209a6b14781179103528093188ed4008"
+}%
+```
+
+The route is not configured because the Fallback Configuration mechanism has excluded the broken `KongPlugin` and its
+dependants, including the `HTTPRoute`, from the configuration.
+
+We can verify this by inspecting the diagnostic endpoint:
+
+```bash
+kubectl port-forward -n kong deploy/kong-controller 8010:10256 &
```

Comment:
Any particular reason to use a different port than already configured in the pod/container : `10256`?

Reply:
Not really, I'll change that to the pod's one.